pcsx2/x86/iR3000A.cpp

```diff
@@ -434,6 +437,11 @@ void _psxFlushCall(int flushtype)
 	_freeX86reg( ecx );
 	_freeX86reg( edx );
 
+	if ((flushtype & FLUSH_PC)/*&& !g_cpuFlushedPC*/) {
```

Comment:
Did you fix a register flush bug here, and/or was this required for the IOP breakpoints?

Reply:
(This is the last review question from my side ;p)

Reply:
This is required for IOP breakpoints otherwise the breakpoint will think it's on the wrong instruction, but this flush type isn't implemented on IOP side for some reason so I just copypasted the code for this from the EE side.